app/test/obituary_test.rb

```diff
@@ -15,7 +15,22 @@ def test_it_returns_a_200
   end
 
   def test_it_returns_formatted_obituaries_json
-  	get '/'
-  	assert_equal 'obituary', JSON.parse(last_response.body).first['data']['type']
-  end
+	  	get '/'
+	  	assert_equal 'obituary', JSON.parse(last_response.body).first['data']['type']
+	end
+
+	def test_it_returns_formatted_covid_obituaries_json
+			get '/covid'
+			assert_equal 'obituary', JSON.parse(last_response.body).first['data']['type']
+	end
+
+	def test_it_returns_formatted_obituary_data_by_name
+			get '/name', name: "George"
+			assert_equal "obituary", JSON.parse(last_response.body).first['data']['type']
+	end
+
+	def test_it_returns_formatted_obituary_data_by_date
+			get '/date', beginning_date: '20170101', ending_date: '20190101'
+			assert_equal "obituary", JSON.parse(last_response.body).first['data']['type']
```

Comment:
Nice. Once we are set on the attributes we are returning, we can come back and make these tests more robust to be sure we are getting the data we want.

Reply:
definitely, looks great for now!